trpc/queries/projectV2.ts

```diff
@@ -465,9 +480,7 @@ export const getUserDashboardProjects = (params: GetUserDashboardProjectsParams)
                     ]),
                 )
                 .where(getGoalsFiltersWhereExpressionBuilder(params.goalsQuery))
-                .where('Goal.archived', 'is not', true)
-                .groupBy('Goal.id')
-                .orderBy(mapSortParamsToTableColumns(params.goalsQuery?.sort, 'Goal', params.activityId)),
+                .where('Goal.archived', 'is not', true),
```

Comment:
groupBy and orderBy will be added in another place?

Reply:
this code is no longer needed here